packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -195,6 +194,10 @@ export default {
       }
     },
 
+    showActions() {
+      return !(this.uploadOrFileCreationBlocked && this.publicPage())
```

Comment:
for the record: I'd like to get rid of the `publicPage` checks.

Reply:
Removed it in CreateAndUpload.vue and two more Vue files, it's now only being used in two action mixings and the upload mixin we might be able to delete when introducing Uppy

Reply:
*replaced it in favor of a `useActiveLocation(isLocationPublicActive, ...)` check